### ISSUE TITLE
[Accessibility]: `Set Selection Anchor` and `Select from Anchor to Cursor` does not work in a11y terminal buffer

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#177029]
CC @meganrogge

The following does not work in a11y terminal buffer. Would you please support them?

* Set Selection Anchor (Ctrl+K, Ctrl+B)

* Select from Anchor to Cursor (Ctrl+K, Ctrl+K)

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!
